airflow/www/views.py

```diff
@@ -1328,7 +1331,7 @@ def set_duration(tid):
         external_logs = conf.get('elasticsearch', 'frontend')
         return self.render_template(
             'airflow/tree.html',
-            operators=sorted({op.__class__ for op in dag.tasks}, key=lambda x: x.__name__),
+            operators=sorted({op for op in dag.tasks}, key=lambda x: x.task_type),
```

Comment:
Do the templates need updating for this change? Previously they were passed classes, now they are passed... something else.

Reply:
Already updated graph.html and tree.html. Use `op.task_type` instead of `op.__name__`.